README.md

```diff
@@ -35,6 +35,43 @@ Eland also provides tools to upload trained machine learning models from common
 
 ## Getting Started
 
+### Install Python
+
+Eland is a Python module, so first of all, you'll need Python installed. Most modern operating systems have Python
+preinstalled. To verify the Python version, run your preferred terminal application and type in the command:
+
+```shell
+python --version
+```
+
+* If `python` command is not available, see if you have `python3`.
+* If you have both `python` and `python3`, pick the one with higher version.
+* If you have either `python` or `python3, but the version is below 3.7, [download and install newer Python](https://www.python.org/downloads/).
+* If you don't have `python` or `python3`, [download and install Python](https://www.python.org/downloads/).
+
+On macOS, you'll need Xcode command-line tools:
+
+```shell
+xcode-select --install
+```
+
+Then, Python 3 can be installed with [homebrew](https://brew.sh/):
+
+```shell
+brew install python
+```
+
```

Comment:
```suggestion
```

This is covered above where readers are instructed to download python from https://www.python.org/downloads/